casper-deploy-notifier/src/types.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Debug)]
+pub struct Notification {
+    pub deploy_hash: String,
+    pub public_key: String,
+    pub success: bool,
+}
```

Comment:
I would suggest to factor out the `DeployProcessed` enum from the `SseData` enum and either use that type directly, or implement the `From` trait

Reply:
Done in https://github.com/cspr-rad/kairos/pull/40/commits/9372b7dd220ed0ec5919cc077c7916436e970654.